ably/rest_presence_spec_integration_test.go

```diff
@@ -315,3 +289,28 @@ func sortPresenceByClientID(items []interface{}) {
 		return items[i].(*ably.PresenceMessage).ClientID < items[j].(*ably.PresenceMessage).ClientID
 	})
 }
+
+func sortData(data string) string {
```

Comment:
`sortData` appears to be a helper function for the integration tests. I have a few comments to make specifically about this function.

1. Firstly, its name is unclear to me. It says it's sorting data, but I can't see how it is sorting data. It's definitely transforming data, but to me sorting is when the order of more than 1 piece of data may or may not change. This function only receives one piece of data so how is it being sorted? Maybe there's a better name for this function, one that's based on it's intent.

2. Reading the logic in this function, the intent is not obviously clear to me, there are also no comments to guide the reader. Are we trying to encode a string to base64 here? What is the purpose of this helper?

3. For any complex logic, we should have an internal unit test. I would like to see a unit test around this helper function that shows some example inputs and outputs. At the moment, when I feed strings into this function I get strings back out. e.g for an input of "100" the output prints as "e" but the output also has a length of 8. (https://go.dev/play/p/fao4XcVOBli) 

Feeding a variety of inputs in sometimes null characters come back, is this intentional?

Reply:
The function is no longer needed and has been removed